spec/controllers/tag_set_nominations_controller_spec.rb

```diff
@@ -0,0 +1,1547 @@
+require 'spec_helper'
+
+describe TagSetNominationsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+
+  let(:tag_set_nomination) { FactoryGirl.create(:tag_set_nomination) }
+  let(:owned_tag_set) { tag_set_nomination.owned_tag_set }
+
+  let(:tag_nominator) { tag_nominator_pseud.user }
```

Comment:
Minor note, these definitions are in reverse order, which is confusing to follow -- ie, would prefer to have tag_nominator_pseud defined first, followed by tag_nominator, ditto for mod_pseud before moderator.